src/main/java/mate/academy/Main.java

```diff
@@ -1,15 +1,34 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
+        MovieSession movieSession = new MovieSession();
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setCapacity(200);
+        movieSession.setMovie(fastAndFurious);
+        movieSession.setCinemaHall(cinemaHall);
+        cinemaHall.setDescription("Anime");
+        CinemaHallService cinemaHallService
+                = (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        cinemaHallService.add(cinemaHall);
+        MovieSessionService movieSessionService
+                = (MovieSessionService) injector.getInstance(MovieService.class);
+        movieSession.setCinemaHall(cinemaHall);
+        movieSessionService.add(movieSession);
```

Comment:
your code is still not tested well enough, we need to test **all** methods from the services you have implemented, like `findAvailableSessions()`, lets call souts for more handy testing, and you will see that you still have problem with your fetching of data